<ui version="4.0" >
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find and Replace</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="findTab" >
      <attribute name="title" >
       <string>Find</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="findPrevB" >
           <property name="text" >
            <string>Find Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="findNextB" >
           <property name="text" >
            <string>Find Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeB" >
           <property name="text" >
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="findWhat" >
           <property name="minimumSize" >
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Find What:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="findEdit" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="replaceTab" >
      <attribute name="title" >
       <string>Replace</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="minimumSize" >
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>Find What:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="findWhatEdit" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="minimumSize" >
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>Replace With:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="replaceWithEdit" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="replacePrevB" >
             <property name="text" >
              <string>Replace Previous</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="replaceNextB" >
             <property name="text" >
              <string>Replace Next</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="close1B" >
             <property name="text" >
              <string>Close</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeB</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>425</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel" >
     <x>331</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close1B</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>412</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
